qlog/src/events/quic.rs

```diff
@@ -571,6 +571,15 @@ pub struct TransportParametersSet {
     pub initial_max_streams_uni: Option<u64>,
 
     pub preferred_address: Option<PreferredAddress>,
+
+    pub unknown_parameters: Vec<UnknownTransportParameter>,
+}
+
+#[serde_with::skip_serializing_none]
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+pub struct UnknownTransportParameter {
+    pub id: u64,
+    pub value: Vec<u8>,
```

Comment:
```suggestion
    pub value: Bytes,
```

Strictly this is a `hexstring` in qlog; see https://quicwg.org/qlog/draft-ietf-quic-qlog-quic-events.html#name-packet_sent and https://quicwg.org/qlog/draft-ietf-quic-qlog-main-schema.html#section-1.2. Basically just a string with some character constraints

We don't do a good job of dealing with that consistently through the codebase (and its a future tasks to do it at somepoint, so not asking you to do it in this PR).  

However, this qlog event should be a stringy type (Bytes is a badly names alias for String in this crate).

When it comes to serializing the actual bytes for the qlog event, we have the `qlog::HexSlice::maybe_string` util function. However, I'm not 100% sure that'll work for your impl of `From`